src/aro/Parser.zig

```diff
@@ -4794,7 +4794,11 @@ const CallExpr = union(enum) {
                 Builtin.tagFromName("__va_start").?,
                 Builtin.tagFromName("va_start").?,
                 => arg_idx != 1,
-                Builtin.tagFromName("__builtin_complex").? => false,
+                Builtin.tagFromName("__builtin_complex").?,
```

Comment:
Are we sure these are memoized properly? Getting a bit worried about something like https://github.com/ziglang/zig/issues/7948 happening here

Reply:
I'm not sure how to know if that's happening - I had to up the eval branch quote for `checkVarArg` - is that why you ask?

Reply:
> is that why you ask?

No just because there are so many calls now. I checked with `@compileLog` and they are getting memoized properly.